Neos.Flow/Classes/Utility/PhpAnalyzer.php

```diff
@@ -82,8 +82,14 @@ public function extractNamespace(): ?string
                     break;
                 }
                 list($type, $value) = $token;
+                if (defined('T_NAME_QUALIFIED') && $type === T_NAME_QUALIFIED) {
```

Comment:
Tokenization of namespaces changed in PHP8 and now provides `T_NAME_(FULLY_)QUALIFIED` instead of a couple of `T_STRING` tokens.